### ISSUE TITLE
Parsing of image messages (different export format now?)

### ISSUE THREAD

**Issue by Stefan-Heimersheim** [noe/whatsapp2md#4]
Hi!

I'm not sure if I understand the parsing of image-messages `if line.startswith('\u200e'):` -- do your image messages all start with a `\u200e` unicode character?

Mine look like
```
12/12/2020, 15:37 - Stefan Heimersheim: IMG-20201212-WA0014.jpg (file attached)
```
which is converted to
```
`15:37` ` Stefan Heimersheim` IMG-20201212-WA0014.jpg (file attached)
```

Maybe there are regional or version differences -- if I get this working I'll send you another PR :)

**Comment by noe** [comment]
Yeah, my image messages all had that unicode character. Given the issues received, I think there are a lot of variations in the format, either derived from differences in the locale, but it also seems that format has changed over time.

Thanks a lot for contributing!

**Comment by Stefan-Heimersheim** [comment]
Turns out the `(file attached)` makes finding images quite easy (assuming no one ends their messages with "(file attached)" ^^) -- I'm testing my version now and I'll send you the PR soon :)

**Comment by noe** [comment]
That part is also dependent on the language of the phone :-(

For instance, in an iphone in Spanish I see this:
```
‎[29/9/20 10:13:49] Noe: ‎<adjunto: 00000030-PHOTO-2020-09-29-10-13-49.jpg>
```

Note that `adjunto` is the Spanish word for `attachment`.

**Comment by Stefan-Heimersheim** [comment]
Yeah I guess you'd just need to add every now version separately when it comes up. Or have an argument to determine the version/format to use.
